src/components/Layout/TableOfContents.jsx

```diff
@@ -16,24 +16,25 @@ const Links = ({ entries }) => (
   </StyledLinkList>
 )
 
-const ChapterList = ({ chapters, entries, title, level = 0 }) => (
+const ChapterList = ({chapters, entries, title, level = 0}) => (
   <StyledChapterList>
     {title && <ChapterListItem key={`${title}${level}`}>
       <ChapterTitle level={level}>{title}</ChapterTitle>
     </ChapterListItem>}
     <ChapterListItem>
-      {entries && <Links entries={entries} />}
+      {entries && <Links entries={entries}/>}
     </ChapterListItem>
     <ChapterListItem>
       {chapters && chapters.map((chapter, index) => (
-        <ChapterList {...chapter} level={level + 1} key={`${index}`} />
+        <ChapterList {...chapter} level={level + 1} key={`${index}`}/>
       ))}
     </ChapterListItem>
   </StyledChapterList>
 )
 
-const TableOfContents = ({ chapters }) => (
-  <TOCWrapper>{chapters.map((chapter, index) => <ChapterList {...chapter} key={index} />)}</TOCWrapper>
+const TableOfContents = ({chapters}) => (
+  <TOCWrapper>{chapters.map((chapter, index) => <ChapterList {...chapter}
+                                                             key={index}/>)}</TOCWrapper>
```

Comment:
This is an example of a very annoying whitespace change. It just bloats up the diff but does not contribute anything meaningful.